opentech/apply/funds/models.py

```diff
@@ -275,6 +339,7 @@ def build_json_order_by(field):
 class ApplicationSubmission(AbstractFormSubmission):
     form_data = JSONField(encoder=DjangoJSONEncoder)
     round = models.ForeignKey('wagtailcore.Page', on_delete=models.CASCADE, related_name='submissions', null=True)
+    user = models.ForeignKey(get_user_model(), on_delete=models.SET_NULL, null=True)
```

Comment:
Should we enforce the user fk? I had it like that initially, but changed it just before the demo to allow empty values.

Reply:
You want to ensure there is no chance the anonymous user could fall through to this stage?

Perhaps we should be doing something more like this: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.SET